osdk/src/commands/new/mod.rs

```diff
@@ -23,7 +23,9 @@ pub fn execute_new_command(args: &NewArgs) {
 /// OSDK assumes that the toolchain used by the kernel should be same same as the toolchain
 /// specified in the asterinas workspace.
 fn aster_rust_toolchain() -> &'static str {
-    include_str!("../../../../rust-toolchain.toml")
+    // Here we can't just include it in the repository root because that can't be
+    // read when publishing. Please ensure update both files when updating the toolchain.
+    include_str!("./rust-toolchain.toml.template")
```

Comment:
This template toml includes the comment "# One should also update ...", which should not appear in an OSDK-powered project.